src/Video.tsx

```diff
@@ -265,6 +266,13 @@ const Video = forwardRef<VideoRef, ReactVideoProps>(
       return VideoManager.setPlayerPauseState(false, getReactTag(nativeRef));
     }, []);
 
+    const capture = useCallback(() => {
+      if (drm) {
+        throw Error('"capture" method can not be called with "drm" prop');
```

Comment:
throwing an exception is a bit violent ? console.warn should be enough ?

Reply:
And BTW, you may defined an empty object in drm... I think you can just remove the test and handle it correctly in native code !